<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeatherDialog</class>
 <widget class="QWidget" name="WeatherDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>194</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Weather Effects</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Time:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Power:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QSlider" name="slider_power">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spin_power">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Weather</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radio_none">
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radio_storm">
        <property name="text">
         <string>Storm</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="radio_snow">
        <property name="text">
         <string>Snow</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QRadioButton" name="radio_rain">
        <property name="text">
         <string>Rain</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QSpinBox" name="spin_time">
       <property name="maximum">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Frames</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radio_none</tabstop>
  <tabstop>radio_rain</tabstop>
  <tabstop>radio_storm</tabstop>
  <tabstop>radio_snow</tabstop>
  <tabstop>slider_power</tabstop>
  <tabstop>spin_power</tabstop>
  <tabstop>spin_time</tabstop>
  <tabstop>button_cancel</tabstop>
  <tabstop>button_ok</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>radio_none</sender>
   <signal>toggled(bool)</signal>
   <receiver>slider_power</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_none</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_power</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_power</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spin_power</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_power</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_power</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
